test/TestUtils.py

```diff
@@ -185,3 +186,36 @@ def test_expand_paths_vars(monkeypatch):
     monkeypatch.setenv('TEST_PATH', test_path)
     assert utils.expand_paths_vars(['~']) == [os.path.expanduser('~')]
     assert utils.expand_paths_vars(['$TEST_PATH']) == [test_path]
+
+
+def test_get_yaml_files(monkeypatch, capsys):
```

Comment:
This should be either converted to use `@pytest.mark.parametrize` or separated into different tests. Currently, it contains too many tests in one. One test should assert one behavior.

Reply:
Couldn't figure out a nice way to utilize `@pytest.mark.parametrize` for this case. Splitted.

Reply:
Right, this'd be tricky to do but probably doable. OTOH we should always try to assess whether it's worth the complications because readability matters.